src/Dialect/ONNX/ONNXOps/Math/DFT.cpp

```diff
@@ -62,10 +73,26 @@ LogicalResult ONNXDFTOpShapeHelper::computeShape() {
   setOutputDims(outputDims);
   return success();
 }
+
+template <>
+LogicalResult ONNXGenericDFTOpShapeHelper<ONNXDFTOp>::computeShape() {
+  typename ONNXDFTOp::Adaptor operandAdaptor(operands, op->getAttrDictionary());
+  IndexExpr axis = createIE->getIntAsSymbol(operandAdaptor.getAxis());
+  return customComputeShape(axis);
+}
+
+template <>
+LogicalResult ONNXGenericDFTOpShapeHelper<ONNXDFTV17Op>::computeShape() {
```

Comment:
Do we really need this when we declared `CONVERTED_TO_SUPPORTED_OPS(ONNXDFTV17Op)` in `ONNXUnsupporteOps.hpp`

Reply:
Okay removed